contracts/token/ERC721/ERC721.sol

```diff
@@ -216,7 +225,7 @@ contract ERC721 is ERC165, IERC721 {
      * @param tokenId uint256 ID of the token being burned
      */
     function _burn(address owner, uint256 tokenId) internal {
-        require(ownerOf(tokenId) == owner);
+        require(ownerOf(tokenId) == owner, "ERC721: burn of unowned token");
```

Comment:
"Unowned" to me sounds like there is no owner. What about...

```suggestion
        require(ownerOf(tokenId) == owner, "ERC721: token burned is not own");
```

Or...

```suggestion
        require(ownerOf(tokenId) == owner, "ERC721: burn of token that is not own");
```

Also affects other similar messages.

Reply:
I'd go for 'that is not own', 'token burned' may imply that the burn did go through.